partest-suite/src/test/resources/scala/tools/partest/scalajs/2.13.0-M2/neg/t7494-no-options.check

```diff
@@ -12,20 +12,22 @@ superaccessors   8  add super accessors in traits and nested classes
     extmethods   9  add extension methods for inline classes
        pickler  10  serialize symbol tables
      refchecks  11  reference/override checking, translate nested objects
-       uncurry  12  uncurry, translate function values to anonymous classes
-        fields  13  synthesize accessors and fields, add bitmaps for lazy vals
-     tailcalls  14  replace tail calls by jumps
-    specialize  15  @specialized-driven class and method specialization
- explicitouter  16  this refs to outer pointers
-       erasure  17  erase types, add interfaces for traits
-   posterasure  18  clean up erased inline classes
-    lambdalift  19  move nested functions to top level
-  constructors  20  move field definitions into constructors
-       flatten  21  eliminate inner classes
-         mixin  22  mixin composition
-        jscode  23  generate JavaScript code from ASTs
-       cleanup  24  platform-specific cleanups, generate reflective calls
-    delambdafy  25  remove lambdas
-           jvm  26  generate JVM bytecode
-       ploogin  27  A sample phase that does so many things it's kind of hard...
-      terminal  28  the last phase during a compilation run
+xplicitinnerjs  12  make references to inner JS classes explicit
+       uncurry  13  uncurry, translate function values to anonymous classes
+        fields  14  synthesize accessors and fields, add bitmaps for lazy vals
+     tailcalls  15  replace tail calls by jumps
+    specialize  16  @specialized-driven class and method specialization
+xplicitlocaljs  17  make references to local JS classes explicit
+ explicitouter  18  this refs to outer pointers
+       erasure  19  erase types, add interfaces for traits
+   posterasure  20  clean up erased inline classes
+    lambdalift  21  move nested functions to top level
+  constructors  22  move field definitions into constructors
+       flatten  23  eliminate inner classes
+         mixin  24  mixin composition
+        jscode  25  generate JavaScript code from ASTs
+       cleanup  26  platform-specific cleanups, generate reflective calls
+    delambdafy  27  remove lambdas
+           jvm  28  generate JVM bytecode
+       ploogin  29  A sample phase that does so many things it's kind of hard...
+      terminal  30  the last phase during a compilation run
```

Comment:
[Conveniently jump backward to the first updated .check files to skip the noise](#diff-a2268a717bb0a4cd30d9c184640ef6f2)